bimvee/importIitYarp.py

```diff
@@ -34,132 +36,208 @@
     For imu:
         - "ts": numpy array of float - seconds
 
-    Can also handle LAE (labelled event) bottles, IMU
+    Can also handle LAE (labelled event) bottles, FLOW (dvs optical flow events), 
+        IMU, SKE (skin), EAR, SKS (skin samples)
```

Comment:
Does it handle only LAE bottles? If not, I would re-phrase accordingly.

I would also highligth the difference between SKE and SKS as follows:
> SKE (skin event) and SKS (skin sample)